src/lib/network/clients/save_network.rb

```diff
@@ -286,12 +288,12 @@ def configure_target
       # set proper network service
       set_network_service
 
-      SCR.Execute(path(".target.bash"), "chkconfig network on")
+      SCR.Execute(path(".target.bash"), "/usr/bin/chkconfig network on")
```

Comment:
i hardly doubt that this is needed at all

Reply:
well, goal of PBI and this PR is not to revisit code. But if you really believe we should remove this line, I can do it.

Reply:
yes please. It is a left over from SysV init times which has no replacement in systemd era (in case of this service)